official-antistasi-community/A3-Antistasi
# 3CB Factions, Add more Templates, New Vehicles, Nodes.

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
## Templates and Selector:
Added Malden Defense Force Template, used as OCC on Malden,
Added Afgan National Army Template, used as OCC on Kunduz,
Added African Desert Army Template, currently unused,
Added Brians HIDF Template with a few changes, used as OCC on Tanoa,
Added [Masai's](https://github.com/Masaigu1) AAF Template with a few changes, used as OCC on Altis.
Added Planes to SetPlaneloadout so they will be used with CASroutine etc.
Corrected a Typo in TKA East Template causing Police sometimes without a Primary Weapon.
Changed CDF's AN2 to a C130 due to Problems with the AN2.

## Logistic Nodes:
Added RBS 70 as a Size 2 Static, it fits most vehicles and doesen't clip that badly.
Added High M240 as a Size 4 Static,
Added a new Pickup Truck from 3CB Factions with 2 Nodes,
Added RHS's Kraz Truck with 5 Nodes.

### Please specify which Issue this PR Resolves.
closes nothing

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Templates should load without any issue,

Check Placement for the new added RBS 70 and the High M240,
Check fitment with other Objects in the new added Pickup Truck

```sqf
"rhs_kraz255b1_cargo_open_vdv" createvehicle getpos (player);
"UK3CB_MDF_B_RBS70" createvehicle getpos (player);
"UK3CB_ADA_B_M240_High" createvehicle getpos (player);
"UK3CB_ADC_C_Pickup" createvehicle getpos (player);
"C_Offroad_01_F" createvehicle getpos (player);
```

********************************************************
Notes:
